smslibrary/src/main/java/com/eis/smslibrary/SMSCore.java

```diff
@@ -16,6 +16,23 @@
  */
 final class SMSCore {
 
+    static SmsManager manager;
+
+    /**
+     * Sets up a given valid custom manager
+     * @param manager The custom manager to set up to send messages
+     */
+    static void setManager(SmsManager manager){
+        SMSCore.manager = manager;
+    }
+
+    /**
+     * Returns the pre-set up manager if not null, else returns the default manager
```

Comment:
Should have placed this in the @return

Reply:
The problem is I like better this 
![FT1W8qeOeG](https://user-images.githubusercontent.com/22344699/70148641-d6a8f280-16a6-11ea-8a68-ad7ac7f19d97.png)
Than this
![5e8qHhwn9s](https://user-images.githubusercontent.com/22344699/70148649-db6da680-16a6-11ea-85b3-74c55f2d820c.png)

Which one do you prefer?

Reply:
Definitely the second one, it's standard and more ordered